rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -301,20 +335,12 @@ static inline uint64_t GetKerningHash(char32_t lchar, char32_t rchar)
 	return (static_cast<uint64_t>(lchar) << 32) | static_cast<uint64_t>(rchar); // 64bit used
 }
 
-static std::shared_ptr<FontFace> GetFontFace(const std::string& fontfile, const int size)
+static std::shared_ptr<FontFace> LoadFontFace(const std::string& fontfile)
 {
 	RECOIL_DETAILED_TRACY_ZONE;
 	assert(CFontTexture::sync.GetThreadSafety() || Threading::IsMainThread());
 	auto lock = CFontTexture::sync.GetScopedLock();
 
-	//TODO add support to load fonts by name (needs fontconfig)
-
-	const auto fontKey = fontfile + IntToString(size);
-	const auto fontIt = fontFaceCache.find(fontKey);
-
-	if (fontIt != fontFaceCache.end() && !fontIt->second.expired())
-		return fontIt->second.lock();
-
 	// get the file (no need to cache, takes too little time)
 	std::string fontPath(fontfile);
 	CFileHandler f(fontPath);
```

Comment:
Pass the `vfsMode` down here:
```suggestion
	CFileHandler f(fontPath, vfsMode);
```